edx_exams/apps/api/v1/views.py

```diff
@@ -179,3 +179,30 @@ def patch(self, request, course_id):
             data = {"detail": "Invalid data", "errors": serializer.errors}
 
         return Response(status=response_status, data=data)
+
+
+class ProctoringProvidersView(APIView):
```

Comment:
You could simplify this by inheriting from DRF's [ListAPIView](https://www.django-rest-framework.org/api-guide/generic-views/#listapiview).

Reply:
Ok nice, i didn't understand this at first, but [this article](https://medium.com/@stevelukis/using-listapiview-unit-test-in-django-rest-framework-5af231ce246) helped me understand it